include/ck/tensor_operation/gpu/thread/threadwise_tensor_slice_transfer_v5r1.hpp

```diff
@@ -315,7 +315,7 @@ struct ThreadwiseTensorSliceTransfer_v5r1
                 forward_sweep_(I0) = true;
 
                 static_for<1, nDim, 1>{}([&](auto i) {
-                    index_t tmp = ordered_dst_access_idx[I0];
```

Comment:
@ozturkosu are we sure this is an implementation error before?

Reply:
@junliume , it was implementation before our small modification.